test/flows/find_coronavirus_support_flow_test.rb

```diff
@@ -0,0 +1,597 @@
+      end
+
+      should "render extra have_you_been_made_unemployed guidance for Scotland" do
+        add_responses nation: "scotland"
+
+        assert_rendered_outcome text: "Get help if you’ve been made redundant (mygov.scot)"
+      end
+
+      should "render self_employed if get_food is yes" do
+        add_responses self_employed: "yes"
+
+        assert_rendered_outcome text: "If you’re self employed, a freelancer, or a sole trader"
+      end
+    end
+
+    context "going_to_work" do
+      setup do
+        add_responses need_help_with: %w[going_to_work],
+                      worried_about_work: "yes",
+                      nation: "england"
+      end
+
+      should "render going_to_work if need_help_with? is going_to_work" do
+        assert_rendered_outcome text: "If you’re worried about going in to work"
+      end
+
+      should "render extra guidance for England" do
+        assert_rendered_outcome text: "What to do if you’ve been told to go in to work and you’re worried (Citizens Advice)"
+        assert_match "https://www.citizensadvice.org.uk/work/coronavirus-if-youre-worried-about-working/", @test_flow.outcome_body
+      end
+
+      should "render extra guidance for Scotland" do
+        add_responses nation: "scotland"
+
+        assert_rendered_outcome text: "What to do if you’ve been told to go back in to work and you’re worried (Citizens Advice Scotland)"
+      end
+
+      should "render extra guidance for Wales" do
+        add_responses nation: "wales"
+
+        assert_rendered_outcome text: "What to do if you’ve been told to go back in to work and you’re worried (Citizens Advice)"
+        assert_match "https://www.citizensadvice.org.uk/wales/work/coronavirus-if-youre-worried-about-working/", @test_flow.outcome_body
+      end
+
+      should "render extra guidance for Northern Ireland" do
+        add_responses nation: "northern_ireland"
+
+        assert_rendered_outcome text: "Get help from Citizens Advice"
+      end
+    end
+
+    context "self_isolating" do
+      setup do
+        add_responses need_help_with: %w[self_isolating],
+                      worried_about_self_isolating: "yes",
+                      nation: "england"
+      end
+
+      should "render self_isolating if need_help_with? is self_isolating" do
+        assert_rendered_outcome text: "If you’re self-isolating"
+      end
+
+      should "render extra guidance for England" do
+        assert_rendered_outcome text: "Apply for a Test and Trace Support Payment"
+      end
+
+      should "render extra guidance for Wales" do
+        add_responses nation: "wales"
+
+        assert_rendered_outcome text: "(Welsh Government)"
+      end
+    end
+
+    context "somewhere_to_live" do
+      setup do
+        add_responses need_help_with: %w[somewhere_to_live],
+                      have_somewhere_to_live: "no",
+                      have_you_been_evicted: "no",
```

Comment:
As per above about that this should also default to yes, given there is a ton of conditionals within it (no need to test them, just nice that the code executes)